### ISSUE TITLE
OSD file recording not working on Goggles v1

### ISSUE THREAD

**Issue by ObiVanDunobi** [bri3d/msp-osd#97]
Hi guys,
first of all, I want to thank you for what you do for the community.

WTFOS OSD works fine for me.
Today I was trying to set recording OSD file to sd card in Goggles V1 but I can't solve it.
I have WTFOS updated to the latest version, the required commands are entered via the CLI, but the OSD file still does not record to the SD card.
The goggles save only standard DJI .srt files.
Please don't you know what the problem might be?
I also noticed with the CLI tab that I can't copy and paste the commands but have to enter them manually.
Could it be related?

Prints from WTFOS Goggles CLI are attached.

Thanks for help
![commands entered](https://user-images.githubusercontent.com/115838121/210602551-4b16273e-283d-4514-a2a6-6d3f5226e824.jpg)
![available options](https://user-images.githubusercontent.com/115838121/210602565-241b33e9-2f61-4145-a5a0-295dc3f028b3.jpg)

**Comment by bri3d** [comment]
Have you tried rebooting your goggles without USB/PC connected, to ensure the SD card is visible to the goggles OS instead of the PC?

**Comment by ObiVanDunobi** [comment]
Yes, I tried several ways, but the osd file did not appear anywhere after
recording the DVR

Dňa ne 22. 1. 2023, 18:35 Brian Ledbetter ***@***.***>
napísal(a):

**Comment by W3srick** [comment]
OSD file recording not working on Goggles v1
I am having this same issue.

**Comment by ysoldak** [comment]
Same here. Goggles V1, no .osd files.
